build.py

```diff
@@ -772,9 +772,10 @@ def package(build_output, pkg_name, version, nightly=False, iteration=1, static=
                                 outfile = new_outfile
                             else:
                                 if package_type == 'rpm':
-                                    # rpm's convert any dashes to underscores
-                                    package_version = package_version.replace("-", "_")
```

Comment:
hmmm this is not the solution I expected. If it works great, it been a while since I touched this code. I believe the reason the `-` are being replaced is so that what comes after the `-` doesn't get understood as the `release` part of the version. 

I am likely wrong here, but testing is king.